istioctl/pkg/util/handlers/handlers_test.go

```diff
@@ -56,6 +56,42 @@ func TestInferPodInfo(t *testing.T) {
 			wantPodName:   "istio-security-post-install-1.2.2-bm9w2",
 			wantNamespace: "istio-system",
 		},
+		{
+			proxyName:     "service/istiod",
```

Comment:
Not sure about this usage and "gateway.gateway.networking.k8s.io/istiod"

Curentl it is `istioctl proxy-config clusters <pod-name[.namespace]> `, right

Reply:
No, current usage is `name.namespace` or `type/name.namespace`. 

The existing bug is type cannot have a `.` when it's perfectly valid syntax in k8s

Reply:
Oh i noticed it, `listener [<type>/]<name>[.<namespace>]` looks very tricky